dts/arm/st/f0/stm32f030Xc.dtsi

```diff
@@ -29,5 +29,41 @@
 			status = "disabled";
 			label = "SPI_2";
 		};
+
```

Comment:
Can you add a comment directly here about the shared IRQ and its consequence (for now)?